packages/x-scheduler/src/joy/event-popover/EventPopover.tsx

```diff
@@ -124,6 +124,7 @@ export const EventPopover = React.forwardRef(function EventPopover(
                         defaultValue={
                           adapter.formatByString(calendarEvent.start, 'yyyy-MM-dd') ?? ''
                         }
+                        aria-describedby={errors?.startDate ? 'startDate-error' : undefined}
```

Comment:
Am I missing something, but it doesn't seem that the conditional id application is necessary here. 🤔 
If the referenced DOM node is empty, the screen reader will not say anything based on such a relationship. 🤔 
```suggestion
                        aria-describedby="startDate-error"
```

Reply:
You're right, fixed!